server/form-pages/apply/reasons-for-placement/basic-information/placementDate.ts

```diff
@@ -21,30 +25,55 @@ type PlacementDateBody = ObjectWithDateParts<'startDate'> & {
 export default class PlacementDate implements TasklistPage {
   title: string
 
+  private inCommunity: boolean
+
   constructor(
     private _body: Partial<PlacementDateBody>,
     public application: ApprovedPremisesApplication,
   ) {
-    const formattedReleaseDate = DateFormats.isoDateToUIDate(
-      retrieveQuestionResponseFromFormArtifact(application, ReleaseDate),
-    )
+    const releaseDate = retrieveOptionalQuestionResponseFromApplicationOrAssessment(application, ReleaseDate)
+
+    if (releaseDate) {
+      const releaseDateObj = DateFormats.isoToDateObj(releaseDate)
+      if (isToday(releaseDateObj) || isFuture(releaseDateObj)) {
+        const formattedReleaseDate = DateFormats.isoDateToUIDate(releaseDate)
+        this.title = `Is ${formattedReleaseDate} the date you want the placement to start?`
+        this.inCommunity = false
```

Comment:
I think calling this `inCommunity` might be a bit confusing. I think it might be better to call it `releaseDatePast`

Reply:
Yeah, that's a good point